src/main/31-cloudfront.tf

```
@@ -16,6 +20,24 @@ module "fe_cdn" {
       domain_name           = module.fe_s3_bucket.s3_bucket_bucket_domain_name
       origin_access_control = "fe_hosting_oac"
     }
+
+    apigw = {
+      origin_id   = var.api_gateway_origin_id
```

```suggestion
      origin_id   = "apigw"
```
I don't expect it to change between environments.